sky/task.py

```diff
@@ -525,9 +527,14 @@ def add_storage_mounts(self) -> None:
         if storage_cloud is None:
             raise ValueError('No available cloud to mount storage.')
         store_type = storage_lib.get_storetype_from_cloud(storage_cloud)
+        return store_type
+
+    def sync_storage_mounts(self) -> None:
+        """Adds storage mounts to the Task."""
```

Comment:
Syncs storage mounts: sync files/dirs to cloud storage.

Reply:
Good catch! Fixed. Thanks!